packages/maestro/src/generator/definitions.ts

```diff
@@ -0,0 +1,40 @@
+// UIP = UI Patterns
```

Comment:
It's better not to abbreviate things for the sake of clarity. In this case, UIPatternFieldTypes is not such a long name.

Reply:
I agree, @GGKapanadze please make this change